CONTRIBUTING.md

```diff
@@ -31,16 +31,18 @@ $ ./gradlew clean dokka :arrow-docs:runAnk
 
 The artifact will reside in `modules/docs/arrow-docs/build/site`. The artifact ΛNK produces could be served as a website by [jekyll](https://jekyllrb.com/), a static site generator.
 
-Jekyll is written in ruby, so you need ruby [gem](https://rubygems.org/pages/download) to install it.
+In order to setup a local version of the documentation in your machine, you will have to install [gem](https://rubygems.org/pages/download) as Jekyll is written in ruby. Afterwards, install [bundler](https://bundler.io/) to manage the project dependencies.
 
 ```
-$ gem install jekyll
+$ gem install bundler
+$ bundle install --gemfile=modules/docs/arrow-docs/Gemfile
 ```
 
-Now you have jekyll, then you can preview the documentation by serving the artifact produced by ΛNK.
+After all the dependencies are installed, you can preview the documentation by serving the artifact produced by ΛNK.
 
 ```
-$ jekyll serve --source modules/docs/arrow-docs/build/site/
+$ cd modules/docs/arrow-docs/
+$ bundle exec jekyll serve --source build/site/
```

Comment:
@calvellido I have tried to run this in my machine... but the local webpage did not start! Could you double check? Not sure if it is something I have done wrong with the latest changes to documentation